### ISSUE TITLE
Using KOKKOS/CUDA segfaults in "threebody" test on tersoff

### ISSUE THREAD

**Issue by akohlmey** [lammps/lammps#272]
running the examples/threebody/in.threebody input with KOKKOS segfaults:
```
[akohlmey@ws18 threebody]$ gdb --args ../../src/lmp_kokkos_cuda_mpich -k on g 1 -sf kk -pk kokkos neigh full newton on -in in.tersoff 
GNU gdb (GDB) Fedora 7.11.1-86.fc24
[...]
Reading symbols from ../../src/lmp_kokkos_cuda_mpich...done.
(gdb) r
Starting program: /home/akohlmey/compile/lammps-icms/src/lmp_kokkos_cuda_mpich -k on g 1 -sf kk -pk kokkos neigh full newton on -in in.tersoff
[...]
LAMMPS (5 Nov 2016)
KOKKOS mode is enabled (../kokkos.cpp:40)
  using 1 GPU(s)
[New Thread 0x7fffe8645700 (LWP 19138)]
[New Thread 0x7fffe7e44700 (LWP 19139)]
[New Thread 0x7fffe7643700 (LWP 19141)]
  using 1 OpenMP thread(s) per MPI task
Lattice spacing in x,y,z = 5.431 5.431 5.431
Created orthogonal box = (0 0 0) to (21.724 21.724 21.724)
  1 by 1 by 1 MPI processor grid
Created 512 atoms
Changing box ...
  orthogonal box = (4.3448 0 0) to (17.3792 21.724 21.724)
  orthogonal box = (4.3448 4.3448 0) to (17.3792 17.3792 21.724)
  orthogonal box = (4.3448 4.3448 4.3448) to (17.3792 17.3792 17.3792)
Reading potential file BNC.tersoff with DATE: 2013-03-21

Thread 1 "lmp_kokkos_cuda" received signal SIGSEGV, Segmentation fault.
LAMMPS_NS::PairTersoffKokkos<Kokkos::Cuda>::setup_params (this=0xe2aa5d0) at ../pair_tersoff_kokkos.cpp:130
130		k_params.h_view(i,j,k).powerm = params[m].powerm;
[...]
(gdb) 
```
This is with gcc 4.9.3, cuda 7.5 on fedora 24.

**Comment by stanmoore1** [comment]
Yes, I was about to create a ticket for this but you beat me to it.

**Comment by stanmoore1** [comment]
I fixed this in my local copy, will be released soon.

**Comment by akohlmey** [comment]
fixed in commit 8756a1017dc8eb46f8917c68b7cac620611d8df4
